src/fsharp/vs/ServiceAssemblyContent.fs

```diff
@@ -254,7 +255,7 @@ module AssemblyContentProvider =
                     | None -> ()
 
                     let thisRequiresQualifierAccess =
-                        if entity.IsFSharp && Symbol.hasAttribute<RequireQualifiedAccessAttribute> entity.Attributes then 
+                        if (entity.IsFSharp && Symbol.hasAttribute<RequireQualifiedAccessAttribute> entity.Attributes) || entity.IsClass then 
```

Comment:
We should count value types and simple types that may have static members as well.

Reply:
I know :) The problem that static members should always be prefixed with the type (any type) name, but module values - if the module is RQA.